addons/website/views/snippets/s_pricelist_cafe.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_pricelist_cafe" name="Pricelist Cafe">
+    <section class="s_pricelist_cafe" data-oe-shape-data="{'shape':'web_editor/Origins/19','flip':[]}">
+        <div class="o_we_shape o_web_editor_Origins_19"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-5 pt64 o_cc o_cc5">
+                    <h2 class="h3-fs">Discover our drinks</h2>
```

Comment:
This will break the display options
```suggestion
                    <h2>
                          <span class="h3-fs">Discover our drinks</span>
                    </h2>
```

Reply:
Good catch, thanks 👍  It's fixed ✅  https://github.com/odoo/odoo/compare/c47fe89675bd4d09ea2fc48fbf5b7858c34395da..d8a0d5f5e64ef4a3ca844d7916ec0d25be07c65b

Reply:
EDIT: as discussed, I removed the span https://github.com/odoo/odoo/compare/d8a0d5f5e64ef4a3ca844d7916ec0d25be07c65b..9103b43598fc4f995df375982e1cdb7d5ffb584c